tests/models/sam/test_processor_tf_sam.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
```

Comment:
Why have a separate test file to test the same class?

Reply:
Also gone now!